### ISSUE TITLE
a11y: warning color not WCAG compliant

### ISSUE THREAD

**Issue by gusnaughton** [AlaskaAirlines/AuroDesignTokens#310]
### What browser client(s) are you seeing the issue on?

- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [x] Desktop
- [x] Mobile

### Steps to reproduce

On our Flight Status web page, we currently have colors to signify early (green), delayed (yellow), or cancelled/diverted flights (red):

<img width="169" alt="Image" src="https://github.com/user-attachments/assets/d08e7169-48bd-4395-a5b8-a3898caad79c" />

<img width="384" alt="Image" src="https://github.com/user-attachments/assets/9b2feb18-9673-4d09-8807-8bb319e33773" />

(mismatched tokens between the flight and the status, bug in current deploy. )

The new "warning" color is very light and not WCAG compliant:

<img width="416" alt="Image" src="https://github.com/user-attachments/assets/2922029e-e8f4-4b6d-88a9-ea3bf48db472" />

<img width="760" alt="Image" src="https://github.com/user-attachments/assets/0373b5e6-7558-4717-bfbb-ffddab7554bc" />

### Reproducible on the Auro doc site

- [ ] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by blackfalcon** [comment]
What tokens is this page using? https://auro.alaskaair.com/color/digital-palette

**Comment by gusnaughton** [comment]
@blackfalcon, there is not a defined color for Light or Dark background "warning" style text (though, there is an error token:)

<img width="638" alt="Image" src="https://github.com/user-attachments/assets/b1bc5809-7471-4bcf-8d9f-e4642b0d83ba" />

The only suitable token I can replace the original delayed color with is ```--ds-basic-color-status-warning```

**Comment by chrisfalaska** [comment]
Hi, @gusnaughton . These appear to be the deprecated v4 tokens.

https://auro.alaskaair.com/getting-started/developers/design-tokens/docs/migration#gsc.tab=0

Here are the new v5 tokens:

https://auro.alaskaair.com/getting-started/developers/design-tokens/alaska-classic

**Comment by chrisfalaska** [comment]
@gusnaughton The recommendation is to use the [Alaska Classic theme](https://auro.alaskaair.com/getting-started/developers/design-tokens/alaska-classic) which features the v5 tokens, with the "classic" colors. That will unlock theming, and provide a seamless update path when it comes time to change to the new [Alaska theme](https://auro.alaskaair.com/getting-started/developers/design-tokens/alaska).

Please let us know if accessibility is still an issue.
